web/src/ApiButton.tsx

```diff
@@ -410,19 +411,40 @@ export function ApiButton(props: React.PropsWithChildren<ApiButtonProps>) {
   }
 }
 
+// UIButtons for a location, sorted into types
+export type ButtonSet = {
+  normal: UIButton[]
+  toggleDisable?: UIButton
+}
```

Comment:
yeah, 'default' works

fwiw, I prefer avoiding terms like "userDefined" since - who's the user? those buttons might come from tilt code, or from extensions, or from a tiltfile author, or from someone who's just running tilt and has never looked at a tiltfile